formats.js

```diff
@@ -5,7 +5,7 @@
  *
  * THIS IS A BACKUP FILE.
  */
- exports.BattleFormats = {
+exports.Formats = [
```

Comment:
for why `Formats`: look at `config/formats.js`
for why `[`: it is really an array, not an object.